src/networking.c

```diff
@@ -2715,6 +2715,7 @@ sds catClientInfoString(sds s, client *client) {
     if (client->flags & CLIENT_UNIX_SOCKET) *p++ = 'U';
     if (client->flags & CLIENT_READONLY) *p++ = 'r';
     if (client->flags & CLIENT_NO_EVICT) *p++ = 'e';
+    if (client->flags & CLIENT_NO_TOUCH) *p++ = 'h';
```

Comment:
Generally the letter has something to do with the flag, so maybe?
```suggestion
    if (client->flags & CLIENT_NO_TOUCH) *p++ = 'T';
```